packages/tax/src/services/tax-module-service.ts

```diff
@@ -136,6 +136,49 @@ export default class TaxModuleService<
     })
   }
 
+  async update(
+    id: string,
+    data: TaxTypes.UpdateTaxRateDTO,
+    sharedContext?: Context
+  ): Promise<TaxTypes.TaxRateDTO>
+  async update(
```

Comment:
I think the code will get simpler if we don't have this particular override, as it's just a special case of the `selector` approach, but it is not as common of an use-case as the `id` signature. WDYT?

Reply:
Is your suggestion to just remove the override and leave the rest or what is your thinking?